packages/cli/src/services/orchestration/main/multi-main-setup.ee.ts

```diff
@@ -24,8 +24,8 @@ export class MultiMainSetup extends TypedEmitter<MultiMainEvents> {
 		super();
 	}
 
-	get instanceId() {
-		return config.getEnv('redis.queueModeId');
+	get hostId() {
```

Comment:
do we really need this getter?

Reply:
I thought it helped readability, but I don't feel strongly about it.

[d86bf91](https://github.com/n8n-io/n8n/pull/11262/commits/d86bf917f66ec928f8709bf563117401077fa2bb)